### PR TITLE
i updated it can i see my family now (Checked with JSONLint)

### PR DESCRIPTION
**Pull request by yodabird19**
Added Fort Sampaloc
Added Veldt Exclusion Zone
Added VL
Added Finkergrad Exclusion Zone
Added The Arch Prison
Added Sneed Sanctuary

### CODE REVIEW

File: exclusion_zones.civmap.json [hunk 1]
```diff
@@ -89,6 +89,22 @@
 				[-1588, -2918]
 			]
 		},
+		{
+			"name": "Fort Sampaloc",
+			"owner": "Lambat",
```
Comment (Gjum) [inline exclusion_zones.civmap.json:94]: add a contact please

File: exclusion_zones.civmap.json [hunk 2]
```diff
@@ -108,6 +124,19 @@
 				]
 			]
 		},
+		{
+			"name": "Rhea Exclusion Zone",
+			"owner": "Columbia",
```
Comment (Gjum) [inline exclusion_zones.civmap.json:129]: add a contact please

File: exclusion_zones.civmap.json [hunk 3]
```diff
@@ -125,13 +154,34 @@
 				[-4651, -56]
 			]
 		},
+		{
+			"name": "Finkergrad Exclusion Zone",
+			"owner": "Kallos",
```
Comment (Gjum) [inline exclusion_zones.civmap.json:159]: add a contact please

File: exclusion_zones.civmap.json (hunk 1 above)
Comment (Gjum) [reply exclusion_zones.civmap.json:94]: still needs contact

Comment (yodabird19) [reply exclusion_zones.civmap.json:94]: I deliberately didn't change this one. Lambat is an active democracy with a rapidly-changing government; I feel it's easier to put "Lambat" than to update this every time the government changes (frequently).

Comment (Gjum) [reply exclusion_zones.civmap.json:94]: can you link to a permanent discord invite then?

File: exclusion_zones.civmap.json [hunk 4]
```diff
@@ -92,7 +92,7 @@
 		{
 			"name": "Fort Sampaloc",
 			"owner": "Lambat",
-			"contact": "Lambat",
+			"contact": "Lambat (https://discord.gg/EYwjerUU3V)",
```
Comment (Gjum) [inline exclusion_zones.civmap.json:95]: I'm worried the closing parenthesis will become part of the url, can you remove the parentheses please?

**Review by Gjum — CHANGES_REQUESTED** [review]

**Review by Gjum — APPROVED** [review]

**Pull request merged**